.github/workflows/dissect-ci.yml

```
@@ -4,4 +4,23 @@ on: [push, pull_request, workflow_dispatch]
 jobs: 
   ci:
     uses: fox-it/dissect-workflow-templates/.github/workflows/dissect-ci-template.yml@main
-    secrets: inherit
\ No newline at end of file
+    secrets: inherit
```

```suggestion

    secrets: inherit
```

The original was missing a newline